prow/config/branch_protection.go

```diff
@@ -323,5 +324,10 @@ func BranchRequirements(org, repo, branch string, presubmits map[string][]Presub
 			optional = append(optional, j.Context)
 		}
 	}
+
+	if inrepoconfigEnabled {
+		// Managed by trigger
+		required = append(required, "prow-config-parsing")
```

Comment:
What is this?

Reply:
This was the presubmit-that's-not-a-presubmit where code in `trigger` posts a status to your PR if the in-repo config you are suggesting is not valid

Reply:
Hmmm, I'd expected us to just keep using something like the `checkconfig` presubmit for this. I understand that a user could delete the `checkconfig` presubmit in their PR to avoid running it, but that would be pretty obvious and intentional.

This might make more sense, but at minimum we shouldn't be doing it for every webhook like Steve said. I think we should discuss this approach and see if we can think of something better before we settle on this.

Reply:
Hmhm, `checkconfig` is actually a very nice idea. The presubmit will need to clone both the code repo and the infra repo but that shouldn't be an issue.